webhook/logparser/v1alpha1/setup.go

```diff
@@ -0,0 +1,12 @@
+package v1alpha1
+
+import (
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+)
+
+func SetupWithManager(mgr ctrl.Manager) {
+	mgr.GetWebhookServer().Register("/validate-logparser", &webhook.Admission{
+		Handler: NewValidatingWebhookHandler(mgr.GetScheme()),
```

Comment:
Since the `NewValidatingWebhookHandler ` func is now only used by the `SetupWithManager` (same package), we can now unexport it, right?